src/components/Details/MinutesItem/MinutesItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from "react";
+import styled from "@emotion/styled";
+
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+const DocumentsListOpen = styled.summary({
+  width: "7.5rem",
+});
+
+const DocumentsListParagraph = styled.p({
+  textDecoration: "underline",
+});
+
+interface Docs {
+  /*Document item label */
+  label: string;
+  /*Document item url */
+  url: string;
+}
+
+interface Items {
+  /*Minute item headline */
+  item: string;
+  /*Array of documents to be shown below*/
+  docs?: Docs[];
+}
+
+export interface MinutesItemProps {
+  minutesItems: Items[];
+}
```

Comment:
Could add a comment for this props field as well?

Reply:
Yes, I tried `listStylePosition: "outside"` and it didn't change the indentation.